recipes/tz/all/test_package/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conan import ConanFile
+from conan.tools.build import cross_building
+
+required_conan_version = ">=2.0.0"
```

Comment:
```suggestion
```
Test package must be compatible with Conan 1.x and 2.x. Most of new features are cross-compatible with both Conan 2.0 and Conan >= 1.53